File: JavaScript/01-Number-Programs/prime.js
function isPrime(num) {
  let count = 0;

  for (let i = 1; i <= num; i++) {
    if (num % i === 0) {
      count++;
    }
  }

  console.log(count === 2 ? "Prime Number" : "Not Prime Number");
}

isPrime(10);

File: 01-Number-Programs/01-Number-Programs/Prime.cs
using System;

namespace _01_Number_Programs
{
    internal class Prime
    {
        void IsPrime(int num)
        {
            if (num <= 1)
            {
                Console.WriteLine(num + " is Not Prime Number");
            }

            int count = 0;

            for (int i = 1; i <= num; i++)
            {
                if (num % i == 0)
                    count++;
            }

            if (count == 2)
                Console.WriteLine(num + " is Prime Number");
            else
                Console.WriteLine(num + " is Not Prime Number");
        }

        public static void Main(string[] args)
        {
            Prime p = new Prime();
            p.IsPrime(7);
        }
    }
}